cmd/osbuild-worker/main.go

```diff
@@ -114,6 +114,10 @@ func main() {
 		log.Fatalf("Could not load config file '%s': %v", configFile, err)
 	}
 
+	if len(config.AcceptedJobOwners) == 0 {
+		config.AcceptedJobOwners = []string{"_weldr"}
```

Comment:
Why not just leave it without an owner? Won't unconfigured workers pick it up then?